vault-core/src/main/java/org/apache/jackrabbit/vault/fs/api/WorkspaceFilter.java

```diff
@@ -146,8 +146,7 @@ void dumpCoverage(@NotNull Node rootNode, @NotNull ProgressTrackerListener liste
             throws RepositoryException;
 
     /**
-     * Dumps the coverage of this filter using the given session. The traversal starts
-     * at the common ancestor of all filter sets. If {@code skipJcrContent} is {@code true}
+     * Dumps the coverage of this filter using the given session. If {@code skipJcrContent} is {@code true}
```

Comment:
Maybe you can also clarify the parameter `listener`. 
Instead of "listener to report progress" rather "listener which receives coverage information"

Reply:
Yup.